### DISCUSSION TITLE
Tensor TriangleMesh Property Usage causes Access violation Error

### DISCUSSION THREAD

**Discussion by gojushin** [isl-org/Open3D#6611] [Q&A]
Heya,

For me, everything that involves Tensor-based TriangleMeshes is severely unstable. 

Take, for example:

```
def some_function(self, tmesh: open3d.cpu.pybind.t.geometry.TriangleMesh):
        verts = tmesh.vertex.positions.numpy()
        vert_vec = o3d.utility.Vector3iVector(verts)  # <--- Crashes here
        return vert_vec
```

Accessing/Using `vert_vec` in any way (sporadically) causes a crash due to an Access violation error (`Process finished with exit code -1073741819 (0xC0000005)`)

(i.e. `print(verts)` also causes the crash.)

I got o3d from Pip and use it without Torch, Tensorflow, or anything, so everything runs on the CPU. 
I had the problem first on 0.17, but it persisted even after upgrading to 0.18.
Numpy is up to date, in case that is relevant.

I'm running the function inside a QThread (although I followed directions from #4923, putting everything in __main__ and even outside of a QThread, which didn't improve anything, sadly). 

Sometimes, randomly, it works; other times, it doesn't. 

Any advice on where else I could look for issues / what could cause the issue?

**Comment by patgadget** [comment]
Hi, i found a solution that worked for me, i downgrade numpy to 1.24.0 and it stop crashing
as found in:
[on stackoverflow](https://stackoverflow.com/questions/78712659/open3d-basic-operations-on-point-clouds-not-working-exited-with-code-322122547)
